spec/javascripts/components/stories/history/history_item_spec.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { shallow } from "enzyme";
+import HistoryItem from "../../../../../app/assets/javascripts/components/stories/History/HistoryItem";
+
+describe('<HistoryItem />', () => {
+  const render = overrideProps => {
+    const defaultProps = {
+      title: 'title',
+      date: '2019/08/27 14:18:00 -0300',
+      user: 'i am user',
+      changes: {}
+    };
+
+    const wrapper = shallow(<HistoryItem {...defaultProps} {...overrideProps} />);
+    const header = wrapper.find('[data-id="history-header"]');
+    const changes = wrapper.find('[data-id="history-changes"]');
+
+    return { wrapper, header, changes };
+  };
+
+  it('renders the component', () => {
+    const { wrapper } = render();
+
+    expect(wrapper).toExist();
+  });
+
+  it('renders header', () => {
+    const { header } = render();
+
+    expect(header).toExist();
```

Comment:
Should some specific text be present in the `header`? Or in `changes`? We should assert it as well :smile:

Reply:
Actually the `header` and `changes` are other components that have her tests too:

https://github.com/Codeminer42/cm42-central/pull/637/files#diff-3c49478dd272270b344271d346564073R1

https://github.com/Codeminer42/cm42-central/pull/637/files#diff-3c49478dd272270b344271d346564073R1

And there, this are tested.